src/sageworks/web_components/experiments/assets/custom_styling.css

```diff
@@ -0,0 +1,28 @@
+
+/*.ag-theme-custom {*/
+/*    --ag-header-height: 30px !important;*/
+/*}*/
+
+.ag-theme-custom .ag-header {
```

Comment:
So this needs to be parameterized, each table will have a different color header bar (see screenshot on https://github.com/SuperCowPowers/sageworks).  So the create_component() would take an arg for the header color like the current data table table class does. If putting in a color/gradient header is too much effort we'll just skip it.

Reply:
I think it wouldn't be that hard, if we create 5 classes like `.ag-theme-custom-blue`, `.ag-theme-custom-red` ...  and use the dag (Dash AG Grid) className as parameter to apply the color we want.